README.md

```diff
@@ -140,15 +140,31 @@ It is possible to use the script `produce_insights_results` from `utils` to prod
 
 ## Testing
 
+The following tests can be run to test your code in `insights-results-aggregator`. Detailed information about each type of test is included in the corresponding subsection:
+
+1. Unit tests: checks behaviour of all units in source code (methods, functions)
+1. REST API Tests: test the real REST API of locally deployed application with database initialized with test data only
+1. Integration tests: the integration tests for `insights-results-aggregator` service
+1. Metrics tests: test whether Prometheus metrics are exposed as expected
```

Comment:
Only number 1? seems OK in preview though

Reply:
yes, markdown can count automatically, which is useful especially for longer lists (when you need to add/remove item at beginning)

Reply:
I guess that this is how "numbered items" work in markdown

Reply:
ah, I see. :+1: